src/BlockTypes/AbstractProductGrid.php

```diff
@@ -1,25 +1,18 @@
 <?php
 /**
- * Set up some shared functionality for grid blocks.
- * NOTE: DO NOT edit this file in WooCommerce core, this is generated from woocommerce-gutenberg-products-block.
```

Comment:
Should we keep this comment or there was a reason to remove it?

Reply:
Since this is in a package now, these comments are not needed as files won't be copied into WooCommerce core directories.